mdjs-layout/src/mdjs-layout.ts

```diff
@@ -213,6 +214,30 @@ export class MdjsLayout extends HTMLElement {
     this.renderIsNavigationShown();
   }
 
+  private get initialColorScheme(): ColorScheme | null {
+    return this.getAttribute('initial-color-scheme') as ColorScheme | null;
+  }
+
+  private set initialColorScheme(value: ColorScheme | null) {
+    if (value === null) {
+      this.removeAttribute('initial-color-scheme');
+    } else {
+      this.setAttribute('initial-color-scheme', value);
+    }
+  }
+
+  private get disableColorSchemeChange() {
+    return this.hasAttribute('disable-color-scheme-change');
+  }
+
+  private set disableColorSchemeChange(value) {
+    if (value) {
+      this.setAttribute('disable-color-scheme-change', '');
+    } else {
+      this.removeAttribute('disable-color-scheme-change');
+    }
+  }
```

Comment:
there is a gotcha here: since we are not using Lit or smth like that, it's not gonna be picked up in the template and rerender, at least I didn't invest into doing this porperly

but, all of these things might become outdated soon, here is why

verbose vanilla is tough vs Lit and similar, I think it's a matter of time when we switch
Lit or not, I'm not sure yet, I'd prefer to not build dockit stuff on Lit, even given Lit 2 has no global side-effects, packd flattens deps, so it's not gonna be working if DS has another major Lit version, which is bad
but Stencil or even smth lightweight is a much better idea, I discarded Stencil due to a few bugs in Backlight, but will return to that ASAP